test/controllers/announcement_controller_test.exs

```diff
@@ -79,7 +79,7 @@ defmodule ConstableWeb.AnnouncementControllerTest do
     assert interest_names == ["boston", "everyone"]
   end
 
-  test "#delete does not work for users who are not the owner of the announcement", %{conn: conn, user: user} do
+  test "#delete does not work for users who are not the owner of the announcement", %{conn: conn} do
```

Comment:
Was this change just to get rid of an unused param warning? Just curious :)

Reply:
Yeah, since I was upgrading versions, I was paying attention to all the warnings and this one was one of them. So I just fixed it here.